### PR TITLE
Add bigmonkey

### PR DESCRIPTION
**Pull request by d1snin**

### CODE REVIEW

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/BigMonkeyCommand.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.kamiblue.botkt.plugins.main.commands
+
+import org.kamiblue.botkt.command.BotCommand
+import org.kamiblue.botkt.command.Category
+
+object BigDogCommand : BotCommand(
```
Comment (5HT2) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/BigMonkeyCommand.kt]: you forgot to change the file name

**Review by 5HT2 — CHANGES_REQUESTED** [review]
You need to register both commands in KamiBluePlugin in order for them to work

**Comment by d1snin** [comment]
done

**Comment by 5HT2** [comment]
> You need to register both commands in KamiBluePlugin in order for them to work

**Comment by d1snin** [comment]
done.

**Review by 5HT2 — APPROVED** [review]

**Pull request merged**